docs/supplement.rst

```diff
@@ -43,68 +116,87 @@ With a `status.yml` file like the following:
     bad:
       77073552: 9
       23434: 10
+    mags:
+      - agasc_id: 1081092600
+        mag_aca: 11.0
+        mag_aca_err: 0.1
```

Comment:
The intent was also to add a specific cat id for a mag_aca that is really a maxmag, right?  Do we need an example for a not-maxmag entry and a maxmag-style entry?

Reply:
Or I'm confused... was the assumption that for all stars with an inferred magnitude they would also have entries in 'bad'?  Did we decide that is why they didn't need catalog ids as a column in 'mags'?

Reply:
I'm not sure I understand.

I thought the only difference in the maxmag case would be that the same agasc_id would also be in the 'bad' section, with a source that identifies maxmag. I don't see where the cat id appears in this picture.

This example is to show a syntactically correct way to write a file. Adding cases like you propose is valid, but it is about semantically correct ways to write the file. The semantics are related to the process and conventions, more than the technical aspect of how to run the scripts.

I would not delay this PR based on semantics. If this is approved before Sunday, I can update the test environment and for Monday we would be using the proposed workflow. We can still tweak the docs based on how we intend to use the file.

Reply:
You answered while I was typing and I just saw it. I think you basically expressed what I understood.

Reply:
Right, I'm using this PR to try to work through more issues than are fully in scope.  I understand that.
With regard to inferred magnitudes, if the intent is that they would also be in the bad section I think we need an additional doc example.